Assets/HoloToolkit/Utilities/Editor/SceneLauncherEditor.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using UnityEngine.SceneManagement;
+
+namespace HoloToolkit.Unity
+{
+    [CustomEditor(typeof(SceneLauncher))]
+    public class SceneLauncherEditor : Editor
```

Comment:
I like this idea. Sorry I haven't had a chance to take a look sooner. I intend to do so soon if I can.